simapp/app.go

```diff
@@ -55,9 +55,9 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/evidence"
 	evidencekeeper "github.com/cosmos/cosmos-sdk/x/evidence/keeper"
 	evidencetypes "github.com/cosmos/cosmos-sdk/x/evidence/types"
-	feegrant "github.com/cosmos/cosmos-sdk/x/feegrant"
+	"github.com/cosmos/cosmos-sdk/x/feegrant"
 	feegrantkeeper "github.com/cosmos/cosmos-sdk/x/feegrant/keeper"
-	feegranttypes "github.com/cosmos/cosmos-sdk/x/feegrant/types"
+	feegrant_m "github.com/cosmos/cosmos-sdk/x/feegrant/module"
```

Comment:
Is this package name okay? shouldn't we consider `feegrantmodule` for this? I can see names like `feegranttypes`, `feegrantkeeper` which are similar cases.

Reply:
I just wanted to make it consistent with @robert-zaremba's PR on authz updates https://github.com/cosmos/cosmos-sdk/pull/9042 but I'm open to change it

Reply:
I would vote for `feegrantmodule` even though it's longer because it's easier to read.